src/components/NurseryWithdrawals/NurseryWithdrawalsFiltersPopover.tsx

```diff
@@ -0,0 +1,260 @@
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}
+        className={classes.popoverContainer}
+      >
+        <div className={classes.popover}>
+          <div className={classes.title}>{strings.FILTERS}</div>
+          <Box className={classes.container}>
+            <Grid container spacing={2}>
+              <Grid item xs={6}>
+                <Typography fontSize='16px' paddingLeft={theme.spacing(2)} color={theme.palette.TwClrBaseGray500}>
+                  {strings.FROM_NURSERY}
+                </Typography>
+                {getAllNurseries(organization).map((n) => (
+                  <Grid item xs={12} key={n.id}>
+                    <Checkbox
+                      id={n.id.toString()}
+                      name={n.name}
+                      label={n.name}
+                      value={hasFilter('fromNurseryIds', n.id)}
+                      onChange={(id, value) => onChangeHandler(id, value, 'fromNurseryIds')}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+              <Grid item xs={6}>
+                <Typography fontSize='16px' paddingLeft={theme.spacing(2)} color={theme.palette.TwClrBaseGray500}>
+                  {strings.PURPOSE}
+                </Typography>
+                {NurseryWithdrawalPurposesValues.map((purpose) => (
+                  <Grid item xs={12} key={purpose}>
+                    <Checkbox
+                      id={purpose}
+                      name={purpose}
+                      label={purpose}
+                      value={hasFilter('purposes', purpose)}
+                      onChange={(id, value) => onChangeHandler(id, value, 'purposes')}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+              <Grid item xs={6}>
+                <Typography fontSize='16px' paddingLeft={theme.spacing(2)} color={theme.palette.TwClrBaseGray500}>
+                  {strings.DESTINATION}
+                </Typography>
+                {plantingSites?.map((plantingSite) => (
+                  <Grid item xs={12} key={plantingSite.id}>
+                    <Checkbox
+                      id={plantingSite.name.toString()}
+                      name={plantingSite.name}
+                      label={plantingSite.name}
+                      value={hasFilter('destinationNames', plantingSite.name)}
+                      onChange={(id, value) => onChangeHandler(id, value, 'destinationNames')}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+              <Grid item xs={6}>
+                <Typography fontSize='16px' paddingLeft={theme.spacing(2)} color={theme.palette.TwClrBaseGray500}>
+                  {strings.SPECIES}
+                </Typography>
+                {species?.map((iSpecies) => (
+                  <Grid item xs={12} key={iSpecies.id}>
+                    <Checkbox
```

Comment:
suggestion - for all the checkboxes, align with top of string?
<img width="232" alt="Terraware App 2022-12-05 08-06-58" src="https://user-images.githubusercontent.com/1865174/205685379-8fdf6dd9-247c-413b-adee-a8d45083fa35.png">

Reply:
yes, ofc, was just pointing this out.